project/Build.scala

```diff
@@ -7,7 +7,8 @@ object BuildSettings {
   val buildSettings = Defaults.coreDefaultSettings ++ Seq (
     organization  := "com.softwaremill.quicklens",
     version       := "1.4.6",
-    crossScalaVersions := Seq("2.11.6", "2.12.0-M3"),
+    scalaVersion  := "2.11.8",
```

Comment:
Without this line SBT falls back to 2.10.x, which prevents the code from compiling :(